tests/python_tests/ov_genai_test_utils.py

```diff
@@ -25,6 +25,7 @@ def get_models_list():
         "microsoft/phi-1_5",
         "microsoft/phi-2",
         "THUDM/chatglm2-6b",
+        "THUDM/chatglm3-6b", # no beam_search
```

Comment:
Does every test pass with `THUDM/chatglm3-6b`? If not, please, mark the failing tests to be skipped. Skips must happen only for that particular model.

Reply:
For for python tests(including beam search) the output is showing `Killed` may be due to resource constraints.

![image](https://github.com/user-attachments/assets/6b447804-358c-4220-b220-796de8959603)
![image](https://github.com/user-attachments/assets/8905e5f2-8f3b-472c-a737-568e3d240875)

Reply:
I refer to python tests, not python samples